config/locales/en.yml

```
@@ -51,3 +51,50 @@ en:
+      csv_units_label: 'Units file (CSV):'
+      csv_capacities_label: 'Capacities file (CSV):'
+      csv_quantities_label: 'Quantities file (CSV):'
+      csv_timewindows_label: 'Timewindows file (CSV):'
+      csv_services_label: 'Services file (CSV):'
+      csv_shipments_label: 'Shipment files (CSV):'
+      json_config_label: 'Configuration file (JSON):'
+      waiting_submit: Waiting for the problem to be sent
+    pickup_delivery:
+      optim_options_legend: 'Optimization options:'
+      optim_duration_label: 'Maximum duration:'
+      optim_minimum_duration_label: 'Minimum duration:'
+      optim_solver_parameter_label: 'Resolution method:'
+      documentation:
+        select_customer_file: 'Select customer file'
+        select_vehicle_file: 'Select vehicle file'
+        params_optim: 'Set your optimization parameters:'
+        params_optim_a: '(a) (optional) The minimum duration (in seconds), the optimization is guaranteed to be run (assuming a feasible instance).'
+        params_optim_b: '(b) (optional) The maximum duration (in seconds) of optimization after which the latest result will be returned.'
+        params_optim_help: 'At least one of the stop conditions (a), (b), (c) must be filled in.'
+        send_click: 'Click "Send"'
+        send_click_a: 'A new submission is sent to the optimization engine.'
+        send_click_b: 'Each new submission is put at the end of a process queue.'
+        send_click_c: 'Each process is divided into several steps: preprocessing, calculation of the matrices, optimization, postprocessing'
+        optim_end: 'Once the optimization is finished, the result of the current optimization appears in a block on the page (you can also download the corresponding CSV file via the link above the block)'
+        optim_end_help: 'The cost of the solution is displayed which allows you to see at a glance if the current solution is better or worse than another one (assuming the problem data stays the same)'
+        import_mt_web: 'Import this csv file into Mapotempo-web (the start and end store information are not imported: remember to define them in Mapotempo-web "beforehand")'
+        import_mt_web_help: 'A schedule is created during import, accessible from the Mapotempo-web "Planning" menu'
+        import_customer_file_columns: 'Your customer file must contain the following columns:'
+        pickup_delivery_setup: 'pickup_setup/delivery_setup: default preparation time, it is only effective for the first activity upon arrival at the linked destination.'
+        import_vehicle_file_columns: 'Your vehicle file must contain the following columns:'
+        setup_multiplier: 'setup_multiplier: a coefficient associated with the vehicle which is multiplied with the pickup_setup and delivery_setup times to calculate the preparation time of the mission (... stays the same ...)'
+        setup_cost: 'setup_cost: cost associated with the setup time.'
+        bold_columns: 'Bold columns are mandatory.'
```

Can we rename the key to be more descriptive
```suggestion
        mandatory_columns: 'Columns in bold are mandatory.'
```

---

`mandatory_columns` left for after